### PR TITLE
Update env.json for v2plugin option (-p)

### PR DESCRIPTION
**Pull request by gkvijay**
fixes https://github.com/contiv/install/issues/141

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Note: The full image contains only Contiv components. Installing Docker Swarm wi
 * Change directories to the extracted folder <br>`cd contiv-$VERSION`
 * To install Contiv with Docker Swarm:<br> `./install/ansible/install_swarm.sh -f cfg.yml -e <ssh key> -u <username> -i`
```
Comment (rhim) [inline README.md]: Can you also update these two as well to say this docker swarm install is only for legacy swarm mode, and is incompatible with native swarm and v2 plugin?

File: README.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ Note: The full image contains only Contiv components. Installing Docker Swarm wi
...
 * To install Contiv with Docker Swarm and ACI:<br> `./install/ansible/install_swarm.sh -f aci_cfg.yml -e <ssh key> -u <username> -i -m aci`
+* To install Contiv v2plugin:<br> `./install/ansible/install_swarm.sh -f aci_cfg.yml -e <ssh key> -u <username> -p`
```
Comment (rhim) [inline README.md]: See above. Please explicitly note docker swarm-mode here.

File: install/ansible/install.sh [hunk 3]
```diff
@@ -131,6 +131,9 @@ cp /var/contiv/key.pem /ansible/roles/auth_proxy/files/
 if [ "$aci_image" != "" ]; then
 	sed -i.bak "s#.*aci_gw_image.*#\"aci_gw_image\":\"$aci_image\",#g" "$env_file"
 fi
+if [ "$contiv_v2plugin_install" == "true" ]; then
```
Comment (rhim) [inline install/ansible/install.sh]: Should it not be exclusive with -i option? AFAIK, there isn't a point in installer covering the docker swarm install and v2plugin install option (even though they can be used together).

File: install/ansible/uninstall.sh [hunk 4]
```diff
@@ -140,6 +140,9 @@ sed -i.bak "s#__CLUSTER_STORE__#$cluster#g" $env_file
 if [ "$aci_image" != "" ]; then
 	sed -i.bak "s#.*aci_gw_image.*#\"aci_gw_image\":\"$aci_image\",#g" "$env_file"
 fi
+if [ "$contiv_v2plugin_install" == "true" ]; then
```
Comment (rhim) [inline install/ansible/uninstall.sh]: How do we set the docker version to ensure that v2plugin is only used with 17.03 or something such? Is it possible for ppl to try v2plugin with an old version of docker?

File: install/ansible/install.sh (hunk 3 above)
Comment (gkvijay) [inline install/ansible/install.sh]: We shouldn't exclude it. veplugin in legacy swarm can be installed. We will not officially support it

File: install/ansible/uninstall.sh (hunk 4 above)
Comment (gkvijay) [inline install/ansible/uninstall.sh]: if docker version is lesser than 17.03, docker plugin install command is not supported and install fails.

File: README.md (hunk 2 above)
Comment (gkvijay) [inline README.md]: We are only installing v2plugin. We are not installing swarm-mode here

File: README.md (hunk 1 above)
Comment (gkvijay) [inline README.md]: will update

File: install/ansible/install.sh (hunk 3 above)
Comment (gkvijay) [inline install/ansible/install.sh]: Made -p and -i as mutually exclusive.

**Comment by rhim** [comment]
LGTM. Go ahead and merge. Please open a pr for doc changes. I think the v2plugin Neelima to be documented better on contiv/install and/or contiv.github.io.

File: install/ansible/install.sh [hunk 5]
```diff
@@ -131,6 +136,9 @@ cp /var/contiv/key.pem /ansible/roles/auth_proxy/files/
...
+	sed -i.bak "s#.*contiv_v2plugin_install.*#\"contiv_v2plugin_install\":\"True\",#g" "$env_file"
+fi
```
Comment (neelimamukiri) [inline install/ansible/install.sh]: Opened https://github.com/contiv/install/issues/149 to handle resetting this for the non-v2 plugin case.
Rest of it LGTM.

**Pull request merged**